<p>
    Shared coding conventions allow teams to collaborate effectively. This rule checks that all file names match a
    provided regular expression.
</p>

<h2>Noncompliant Code Example</h2>
With default regular expression: <code>^[a-z][-A-Za-z0-9]*.feature$</code>
<pre>
MyFeature.feature
</pre>

<h2>Compliant Solution</h2>
With default regular expression: <code>^[a-z][-A-Za-z0-9]*.feature$</code>
<pre>
my-feature.feature
</pre>
