<p>
    Shared coding conventions allow teams to collaborate effectively. This rule checks that all tags match a provided
    regular expression.
</p>

<h2>Noncompliant Code Example</h2>
With default regular expression: <code>^[a-z][-a-z0-9]*$</code>
<pre>
@MyTag
</pre>

<h2>Compliant Solution</h2>
With default regular expression: <code>^[a-z][-a-z0-9]*$</code>
<pre>
@mytag
@my-tag
</pre>
